gce-containers-startup/utils/volumes.go

```diff
@@ -146,5 +152,193 @@ func processHostPathVolume(volume *api.HostPathVolume) (VolumeHostPathAndMode, e
 }
 
 func processGcePersistentDiskVolume(volume *api.GcePersistentDiskVolume) (VolumeHostPathAndMode, error) {
-	return VolumeHostPathAndMode{}, fmt.Errorf("Unimplemented.")
+	if volume.FsType != "" && volume.FsType != ext4FsType {
+		return VolumeHostPathAndMode{}, fmt.Errof("Unsupported filesystem type: %s", volume.FsType)
```

Comment:
Maybe:
"Unsupported filesystem type for PD: %s" in case a user provides more than one volume?